### PR TITLE
[FEATURE: UIPT-63] - Zoom

### PR DESCRIPTION
**Pull request by andriiveremchuk**

### CODE REVIEW

File: src/app/components/product-preview/product-preview.component.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, Input, OnInit, HostListener, ViewChild, ElementRef, DoCheck } from '@angular/core';
+
+import { NgxImageZoomComponent } from 'ngx-image-zoom';
 
 import { Store } from '@ngrx/store';
```
Comment (IrynaMukhina) [inline src/app/components/product-preview/product-preview.component.ts]: please move all store imports to one place

File: src/app/components/product-preview/product-preview.html [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
   </ng-template>
-  <div *ngFor="let media of mediaSource; let i = index"
+  <div
+    *ngFor="let media of mediaSource; let i = index"
     class="product-preview__small-view__container"
-    (click)="onImageSelect(i)">
-    <img *ngIf="!media.video; else videoSmall"
+    (click)="onImageSelect(i);"
+  >
+    <img
```
Comment (TorskyM) [inline src/app/components/product-preview/product-preview.html]: Align please

File: src/app/components/product-preview/product-preview.scss [hunk 3]
```diff
@@ -2,6 +2,31 @@
...
+}
+
+::ng-deep .ngxImageZoomContainer > div { 
+  width: auto;
+  height: auto;
+}
+
+::ng-deep .ngxImageZoomContainer > img {
+    height: auto;
+    width: 100%;
```
Comment (IrynaMukhina) [inline src/app/components/product-preview/product-preview.scss]: please align

File: src/app/components/product-preview/product-preview.scss [hunk 4]
```diff
@@ -2,6 +2,31 @@
...
+  max-width: 100%;
+  max-height: 100%;
+
+  &:hover {
+    cursor: zoom-in;
+  }
+}
+
+::ng-deep .ngxImageZoomContainer > div { 
+  width: auto;
```
Comment (TorskyM) [inline src/app/components/product-preview/product-preview.scss]: Can you use nesting here?

**Pull request merged**